### PR TITLE
Added missing options #141, #143, #144

### LINKED ISSUE #141: Missing "Subtitle" option
There is a missing `Subtitle` option under plugins. Please refer to https://www.chartjs.org/docs/latest/configuration/subtitle.html

It has exactly the same configuration options with the main [title](https://www.chartjs.org/docs/latest/configuration/title.html). So it can be used the `Title` class for this.

### LINKED ISSUE #143: Missing PointStyle
`rectRounded` is missing

### LINKED ISSUE #144: Missing pointStyle option in BubbleDataset
Screenshot 2024-05-10 at 11 44 32

### PR DESCRIPTION
**Pull request by aripddev**
Fixes #141
Fixes #143
Fixes #144

### CODE REVIEW

**Comment by aripddev** [comment]
#143 Added the missing `rectRounded` in PointStyle.

https://www.chartjs.org/docs/latest/configuration/elements.html#point-styles

File: chartjs-java-model/src/main/java/software/xdev/chartjs/model/options/Plugins.java [hunk 1]
```diff
@@ -41,6 +42,22 @@ public Plugins setTitle(final Title title)
...
+	 * @see #setSubtitle(Title)
+	 */
+        public Title getSubtitle() {
+          return this.subtitle;
+        }
+
+	/**
+	 * The subtitle configuration is passed into the options.subtitle namespace. The global options for the chart subtitle is
+	 * defined in Chart.defaults.global.subtitle.
+	 */
```
Comment (melloware) [inline chartjs-java-model/src/main/java/software/xdev/chartjs/model/options/Plugins.java:56]: Looks like your Javadoc formatting is off?

**Review by melloware — CHANGES_REQUESTED** [review]

File: chartjs-java-model/src/main/java/software/xdev/chartjs/model/options/Plugins.java (hunk 1 above)
Comment (aripddev) [reply chartjs-java-model/src/main/java/software/xdev/chartjs/model/options/Plugins.java:57]: I use Ctrl+Shift+F to format the code and javadoc, but it changes the format completely. So, I made it manually :) Is there a better way to do it?

Comment (melloware) [reply chartjs-java-model/src/main/java/software/xdev/chartjs/model/options/Plugins.java:57]: Yeah you did the right thing not reformatting the whole file i think fixing it by hand is fine.

**Comment by AB-xdev** [comment]
Thank you for the PR and the issues :)

LGTM, however codestyle is broken.

I will do a quick reformat.

File: chartjs-java-model/src/main/java/software/xdev/chartjs/model/options/Plugins.java (hunk 1 above)
Comment (AB-xdev) [reply chartjs-java-model/src/main/java/software/xdev/chartjs/model/options/Plugins.java:58]: > I use Ctrl+Shift+F to format the code and javadoc, but it changes the format completely. So, I made it manually :) Is there a better way to do it?

Have a look at our [CONTRIBUTING.md](https://github.com/xdev-software/chartjs-java-model/blob/develop/CONTRIBUTING.md#developing). Once setup correctly your IntelliJ does everything for you.

**Pull request merged**